core/hosting/HostWithCoreClrHost/build.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+SET SRCDIR=%~dp0src
+SET OUTDIR=%~dp0bin\windows
+
+mkdir %OUTDIR%
+
+REM Build managed component
+echo Building Managed Library
+dotnet publish --self-contained -r win10-x64 %SRCDIR%\ManagedLibrary\ManagedLibrary.csproj -o %OUTDIR%
+
+REM Build native component
+cl.exe %SRCDIR%\SampleHost.cpp /Fo%OUTDIR%\ /Fd%OUTDIR%\SampleHost.pdb /EHsc /Od /GS /sdl /Zi /D "WINDOWS" /link /out:%OUTDIR%\SampleHost.exe
```

Comment:
This script is assuming build is on a Windows 10 x64 machine and that the corresponding cl.exe is on the path.

It might be nice to remove or document that assumption.

In the last month I read a news bite which mentioned Windows 10 surpassed Windows 7.  So they are currently roughly even.

Reply:
Agreed. Or you could document this script must be run from a Visual Studio Developer command prompt.

Reply:
Good point. I'll mention that the build script builds for Win10 and assumes cl.exe is on the path (and that a VS developer command prompt will have the correct path set).